src/Controls/src/Core/HandlerImpl/Toolbar/Toolbar.Android.cs

```diff
@@ -31,8 +31,6 @@ public partial class Toolbar
 				if (_platformTitleView != null)
 					_platformTitleView.Child = null;
 
-				_platformTitleViewHandler?.DisconnectHandler();
```

Comment:
I see these have been removed, is this not needed or we don't need these events to fire anymore?

Reply:
There are a couple annoying players here. 

1) `TitleView` could probably use a `Templated` version so a new xplat view gets created for different `Toolbar` instances
2) It's annoying that `Shell` creates different `Toolbars` in the first place. Like, on shell when you push a page it recreates an entire coordinatorlayout with a new toolbar. When you navigate through tabs it creates a new toolbar for each tab. 

So, the `TitleView` might get reused if you're specifying the `TitleView` on the `Shell` itself, then the view/handler will get moved to a different `Toolbar` so we don't want to disconnect the handler here.